ddtrace/tracer/span.go

```diff
@@ -337,6 +338,27 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+// Format implements fmt.Formatter.
+func (s *span) Format(f fmt.State, c rune) {
+	switch c {
+	case 's':
+		fmt.Fprint(f, s.String())
+	case 'v':
+		if tr, ok := internal.GetGlobalTracer().(*tracer); ok {
```

Comment:
I'm not super happy with this, but this will be the most consistent way to get the application's service name.

Reply:
I understand. I'm guessing the overhead is pretty low though?

Reply:
Yes, overhead is low. We already do this in `(*spancontext).finish()`

Reply:
I think this is fine Kyle. We want to get the active tracer. Who knows what wizardry some user might pull and have another tracer than the one which created this span.

But... What if there is _no_ tracer? Let's have a fallback. First, try `globalconfig.ServiceName`, second, use the span's service. WDYT?

Reply:
I don't think we can use the span's service. I think, thankfully, that the value of `globalconfig.ServiceName()` should always be what we want to print for service.